public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+:marked
+  Here,`child-content` is just a directive that defines content:
+  
++makeExample('component-communication/ts/fragments/contentchildren-intro.ts', 'child-content')
+
+:marked
+  We'd like to render a view so that a particluar component, ie. `ChildComponent`, is used for each `child-content`:
+  
++makeExample('component-communication/ts/fragments/contentchildren-intro.ts', 'child-comp')
+
+:marked
+  Somehow, we need to transpose the `child-content` elements into `parent-comp` as `ChildComponent` instances, and want to communicate
+  from the parent component to its children. We know that input data binding is a simple and great way to implement parent-to-child 
+  communication so that we'd like to apply a template like this, where `myContent` represents the collection of `child-content` elements we
+  nested into `parent-comp`:
+  
++makeExample('component-communication/ts/fragments/contentchildren-intro.ts', 'parent-comp-def', null, {pnk: /(myContent)/g})
+
+:marked
+  The issue is that we do not know how to get a collection of `child-content` elements and store them in `myContent` so that we
+  can iterate through this collection, define the child component instances, and establish the input property binding.
+  
+  Angular provides the `@ContentChildren` annotation we can use to define that we need to get a collection of children that are used
+  as content within our parent component. The framework takes care of initializing this collection during component instantiation, 
+  detects changes, and dynamically updates the collection of children:
+  
++makeExample('component-communication/ts/fragments/contentchildren-intro.ts', 'parent-comp', null, {pnk: /(@ContentChildren)/g})
+
+:marked
+  [Jump to details](#-6-parent-to-child-communication-where-children-are-transposed-in-mdash-using-contentchildren-)
+
+  ### The Hero Job Board sample
+  
+  In this chapter, we will build a sample application that help us involve our heroes in actions when we really need their help.
+  In this app, users can act on behalf of the person in need, and also on behalf of each hero. The basic workflow is the following:
+  1. The person in need invites heroes to listen to job requests.
+  1. When there's a decent job that requires superpower, the person in need announces a job request that is received by all heroes.
+  1. Heroes can opt to undertake the job.
+  1. When having received applications for the job, the person in need assigns the job to the one of the responding heroes.
+  1. The assigned hero is notified about winning the job, the others receive a message about losing the opportunity.
+  
+  Here is a screenshot of our complete application:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/app-ui-snapshot.png" alt="UI in action")
+  
+:marked
+  ### Live Example
+  
+  The application is available as a live example. It is a single Angular app that uses the router configured with separate routing paths 
```

Comment:
I don't think that we are clear enough about what this sample is for.

We should say that we're implementing the same example 6 different ways, each way illustrating one of the aforementioned techniques.

The role of the router is to make it easy for us to hop among the 6 different illustrations.  We should also correlate the 6 techniques with the 6 illustrations and make that clear in the sample links.

Also note that we have 6 techniques and 7 sample links. That doesn't align.